src/main/kotlin/net/axay/blaubot/commands/Bingus.kt

```diff
@@ -0,0 +1,21 @@
+package net.axay.blaubot.commands
+
+import dev.kord.common.annotation.KordPreview
+import dev.kord.core.behavior.followUp
+import dev.kord.core.entity.interaction.Interaction
+import net.axay.blaubot.commands.api.SlashCommand
+
+@KordPreview
+object Bingus: SlashCommand(
+    "bingus",
+    "get bingus"
+) {
+    override suspend fun handleCommand(interaction: Interaction) {
+        interaction.acknowledge(true).followUp {
+            embed {
```

Comment:
Add more information about Bingus to the embed (an image alone is not enough)

Reply:
i can do this

Reply:
please add more information about bingus himself